classification1.Rmd

```diff
@@ -206,28 +200,28 @@ changing them to "Benign" and "Malignant" using the `labels` argument.
 ```{r 05-scatter, fig.height = 4, fig.width = 5, fig.cap= "Scatter plot of concavity versus perimeter coloured by diagnosis label"}
 perim_concav <- cancer %>%
   ggplot(aes(x = Perimeter, y = Concavity, color = Class)) +
-  geom_point(alpha = 0.5) +
-  labs(color = "Diagnosis") +
+  geom_point(alpha = 0.6) +
+  labs(color = "Diagnosis", x = "Perimeter (standardized)", y = "Concavity (standardized)") +
   scale_color_manual(labels = c("Malignant", "Benign"), values = c("orange2", "steelblue2"))
 perim_concav
 ```
 
 In Figure \@ref(fig:05-scatter), we can see that malignant observations typically fall in
-the the upper right-hand corner of the plot area. By contrast, benign
-observations typically fall in lower left-hand corner of the plot. In other words, benign observations 
-tend to have lower concavity and perimeter values, and malignant ones tends to have larger 
-concavity and perimeter values. Suppose we
+the upper right-hand corner of the plot area. By contrast, benign
+observations typically fall in the lower left-hand corner of the plot. In other words,
+benign observations tend to have lower concavity and perimeter values, and malignant
+ones tend to have larger values. Suppose we
 obtain a new observation not in the current data set that has all the variables
 measured *except* the label (i.e., an image without the physician's diagnosis
-for the tumour class). We could compute the perimeter and concavity values,
+for the tumor class). We could compute the standardized perimeter and concavity values,
 resulting in values of, say, 1 and 1. Could we use this information to classify
```

Comment:
I think we might want to suggest a conclusion for each of these. For example, you could say just purely intuitively "well if the values are 1 and 1, the point would lie in the middle of the orange cloud. it's probably orange then"

Reply:
and maybe reduce the number of these